quiz.rb

```diff
@@ -2,16 +2,47 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+ def initialize(name,breed,age)
+  @name = name
+  @breed = breed
+  @age = age
+     attr_reader :name
+     attr_reader :breed
+     attr_reader :age
+   end
+
+
 # Question 2
 # Create a new Dog object.
 
+def treats
+        puts "loves bones"
+    end
+
+
 # Question 3
 # Create a Person class, with attributes name, age, and location.
+
+class Person
+ def initialize(name,age,location)
+  @name = name
+  @age = age
+  @location = location
+
+  attr_reader :age
+  attr_writer :location
+  attr_writer :name
+
+end
+
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
 # Question 4
 # Create a new Person object using parameters "Dave", 32, and "Ohio".
+Dave = Person.new("Dave", 32, "Ohio")
+
 # Then, set that Person's location to "Somerville"
 
 # Question 5
```

Comment:
I know it's hard, but you should try to always submit something for each question, even if you're not sure; you might know it better than you think.